sdk/core/azure-core/test/ut/transport_adapter_base_test.hpp

```diff
@@ -23,6 +23,10 @@ namespace Azure { namespace Core { namespace Test {
   namespace _detail {
     constexpr static const char AzureSdkHttpbinServerSchema[] = "https";
     constexpr static const char AzureSdkHttpbinServer[] = "azuresdkforcpp.azurewebsites.net";
```

Comment:
Should this logic be moved into a single source rather than being defined in two different places?

Reply:
Yeah, that's a good suggestion. I will see how to remove the existing duplication here.